galleries/examples/pie_and_polar_charts/polar_demo.py

```diff
@@ -4,21 +4,41 @@
 ==========
 
 Demo of a line plot on a polar axis.
+
+The second plot shows the same data, but with the radial axis starting at r=1
+and the angular axis starting at 0 degrees and ending at 225 degrees. Setting
+the origin of the radial axis to 0 allows the radial ticks to be placed at the
+same location as the first plot.
 """
 import matplotlib.pyplot as plt
 import numpy as np
 
 r = np.arange(0, 2, 0.01)
 theta = 2 * np.pi * r
 
-fig, ax = plt.subplots(subplot_kw={'projection': 'polar'})
+fig, axs = plt.subplots(2, 1, figsize=(5, 8), subplot_kw={'projection': 'polar'},
+                        layout='constrained')
+ax = axs[0]
 ax.plot(theta, r)
 ax.set_rmax(2)
-ax.set_rticks([0.5, 1, 1.5, 2])  # Less radial ticks
+ax.set_rticks([0.5, 1, 1.5, 2])  # Fewer radial ticks
 ax.set_rlabel_position(-22.5)  # Move radial labels away from plotted line
 ax.grid(True)
 
 ax.set_title("A line plot on a polar axis", va='bottom')
+
+ax = axs[1]
+ax.plot(theta, r)
+ax.set_rmax(2)
+ax.set_rmin(1)  # change the radial axis to only go from 1 to 2
+ax.set_rorigin(0)  # Set the origin of the radial axis to 0.
```

Comment:
```suggestion
ax.set_rmin(1)  # Change the radial axis to only go from 1 to 2
ax.set_rorigin(0)  # Set the origin of the radial axis to 0
```
Tiny nit, but the rest of the comments have capital letters and no periods.